Rubberduck.Parsing/Annotations/AnnotationType.cs

```diff
@@ -64,7 +64,11 @@ public enum AnnotationType
         PredeclaredId = 1 << 16 | Attribute | ModuleAnnotation,
         [AttributeAnnotation("VB_Exposed", "True")]
         Exposed = 1 << 17 | Attribute | ModuleAnnotation,
-        Obsolete = 1 << 18 | MemberAnnotation | VariableAnnotation
+        Obsolete = 1 << 18 | MemberAnnotation | VariableAnnotation,
```

Comment:
Shouldn't this have a `Flags` annotation? I'd need to check, but this also might warrant a unit test that there aren't any overlapping flags. See `RubberduckTests.FlagSanityTests`

Reply:
This enum is a bit strange in that it both has flags and regular values. Those top section are the flags and the bottom section are the regular vales not to be used as flags.

Reply:
I thought that was the purpose of `[Flags]` - if you have a variable typed as `AnnotationType`, it holds multiple values (all of the set bits). For example, `var foo = AnnotationType.ModuleDescription` is not a discrete value - it is also `AnnotationType.Attribute` and `AnnotationType.ModuleAnnotation`. Defining members with their own bit in combination with others only means that they require other flags to be set. I.e., if `foo.HasFlag(AnnotationType.ModuleAnnotation)`, it's implicitly true that it has the other flags based on the definition. They're more like convenient aliases for equity checks or assignments. I generally call them out as `Flags` any time the enum is used with `HasFlag`, but I guess that's more of a style preference.